Makefile

```diff
@@ -27,7 +27,7 @@ T_YELLOW := \e[0;33m
 T_RESET := \e[0m
 
 .PHONY: all
-all: 1.10 1.11 1.12
+all: 1.10 1.11 1.12 1.14
```

Comment:
Should we add 1.15 since we want to upgrade to that at some point?

Reply:
yes - working on to add 1.15